src/task-actions.ts

```diff
@@ -538,6 +539,8 @@ export async function starknetTestAction(
     hre: HardhatRuntimeEnvironment,
     runSuper: RunSuperFunction<TaskArguments>
 ) {
+    const cached = await handleCache(args, hre);
```

Comment:
This looks like: "if nothing was cached, exit with status code 1", which is probably not what we want here.